egov-dss-dashboards/dashboard-analytics/ChartApiConfig.json

```diff
@@ -31284,5 +31284,193 @@
     "filterKeys": [{ "key": "village" }],
     "insight": {},
     "_comment": " "
+  },
+  "incompleteRecordsByDistrict":{
+    "chartName": "DSS_HEALTH_INCOMPLETE_HOUSES",
+    "queries": [
+      {
+        "module": "COMMON",
+        "dateRefField": "Data.createdTime",
```

Comment:
household index doesn't have this field, can you please change it to Data.household.auditDetails.createdTime